README.md

```diff
@@ -55,5 +55,8 @@ For all other questions, please use [GitHub discussions](https://github.com/awsl
 We enforce a style using [Google checkstyle](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml).
 This can be verified in the tests; tests will not pass with checkstyle errors.
 
+## Aurora Engine Version Testing
+This driver has been tested against MySQL 5 and MySQL 8s.
```

Comment:
This might be unintentionally misleading. Can we explicitly state the MySQL versions we tested against and the Aurora engine versions in our integration pipelines?